<ui version="4.0" >
 <class>jewelryGoodsFilesAddClass</class>
 <widget class="QWidget" name="jewelryGoodsFilesAddClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>460</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <family>宋体</family>
    <pointsize>16</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle" >
   <string>jewelryGoodsFilesAdd</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QWidget" native="1" name="widget" >
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>商品条码：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="leCommodityID" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>商品名称：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="leCommodityName" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>类别：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="leCommodityType" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>商品单位：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="lbCommodity" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>商品质量：</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>备注：</string>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="7" column="1" >
       <widget class="QTextEdit" name="textEdit" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
       </widget>
      </item>
      <item row="8" column="0" >
       <spacer name="verticalSpacer" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>199</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="label" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text" >
         <string>商品折扣</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="QDoubleSpinBox" name="dbCommodityDiscounts" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QDoubleSpinBox" name="dbCommodityPrice" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QComboBox" name="cbCommodityUnit" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <item>
         <property name="text" >
          <string>克</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>颗</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="2" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>克</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="lbCertificate" >
        <property name="text" >
         <string>证书编号：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QLineEdit" name="leCertificate" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QWidget" native="1" name="widget_2" >
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbOK" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbCancel" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>leCommodityID</tabstop>
  <tabstop>leCommodityName</tabstop>
  <tabstop>leCommodityType</tabstop>
  <tabstop>dbCommodityPrice</tabstop>
  <tabstop>dbCommodityDiscounts</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>pbOK</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
